src/debugpy/_vendored/pydevd/tests_python/test_debugger_json.py

```diff
@@ -123,13 +123,15 @@ def write_list_threads(self):
     def wait_for_terminated(self):
         return self.wait_for_json_message(TerminatedEvent)
 
-    def wait_for_thread_stopped(self, reason='breakpoint', line=None, file=None, name=None):
+    def wait_for_thread_stopped(self, reason='breakpoint', line=None, file=None, name=None, preserve_focus_hint=None):
```

Comment:
Changes on `test_debugger_json.py` are unrelated to this issue (it seems that was missing from the previous pull request, so, just catching up here).